### ISSUE TITLE
When navigating to an article page on web, focus isn't moved to the article component so keyboard navigation doesn't work

### ISSUE THREAD

**Issue by jacksonh** [omnivore-app/omnivore#752]
Can replicate on web starting on library navigate to an article with arrow keys, hit enter, attempt to scroll with arrow keys. Note how scrolling with arrow keys and space doesn't work.
